app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -73,69 +79,49 @@ class InstallerViewModel(
     private var inputFile: File? = null
 
     private var installedApp: InstalledApp? = null
-    var isInstalling by mutableStateOf(false)
+    internal var isInstalling by mutableStateOf(false)
         private set
-    var installedPackageName by mutableStateOf<String?>(null)
+    internal var installedPackageName by mutableStateOf<String?>(null)
         private set
-    val appButtonText by derivedStateOf { if (installedPackageName == null) R.string.install_app else R.string.open_app }
+    internal val appButtonText by derivedStateOf { if (installedPackageName == null) R.string.install_app else R.string.open_app }
```

Comment:
```suggestion
    val appButtonText by derivedStateOf { if (installedPackageName == null) R.string.install_app else R.string.open_app }
```
Is internal necessary?